code/modules/surgery/organs/tongue.dm

```diff
@@ -7,8 +7,20 @@
 	slot = ORGAN_SLOT_TONGUE
 	attack_verb_continuous = list("licks", "slobbers", "slaps", "frenches", "tongues")
 	attack_verb_simple = list("lick", "slobber", "slap", "french", "tongue")
+	/**
+	 * A cached list of paths of all the languages this tongue is capable of speaking
+	 *
+	 * Relates to a mob's ability to speak a language - a mob must be able to speak the language
+	 * and have a tongue able to speak the language (or omnitongue) in order to actually speak said language
+	 */
 	var/list/languages_possible
```

Comment:
Seems like a very good candiate for VAR_PRIVATE so people don't overwrite this instead of the get proc